src/CLI/actioner/sonic-cli-pfm.py

```diff
@@ -65,14 +65,18 @@ def run(func, args):
 
         else :
            api_response = getattr(aa,func.__name__)(*keypath)
-
+     
         if api_response is None:
             print ("Success")
         else:
-            response = api_response.to_dict()
-            value =  response['openconfig_platformcomponents']['component'][0]['state']
-            if value is None:
+            api_response = aa.api_client.sanitize_for_serialization(api_response)
+            value =  api_response['openconfig-platform:components']['component'][0]['state']
+	    if value is None:
                 return
+	    if 'oper-status' in value:
+		temp = value['oper-status'].split(':')
+		if temp[1] is not None:
```

Comment:
Do you need to take the length of the temp list and use the last entry int the list right?

Reply:
I will update it.